drivers/flash/Kconfig.rpi_pico

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2022 Yonatan Schachter
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config FLASH_RPI_PICO
+	bool "Raspberry Pi Pico flash driver"
+	default y
+	depends on FLASH && SOC_SERIES_RP2XXX
```

Comment:
`depends on DT_HAS_RASPBERRYPI_PICO_FLASH_CONTROLLER_ENABLED`  should replace the 'depends on FLASH && SOC_SERIES_RP2XXXX`